package.json

```diff
@@ -29,38 +35,48 @@
     "test:e2e:opera": "mocha --opera",
     "test:e2e:safari": "mocha --safari",
     "test:server:start": "http-server &",
-    "release": "np",
-    "release:prepare": "npm run release:prepare:build && npm run release:prepare:css && npm run release:prepare:commit",
-    "release:prepare:build": "npm run build",
-    "release:prepare:css": "replace '\\.\\./style.css' '/node_modules/vl-ui-map/style.css' vl-map.js",
-    "release:prepare:commit": "git add -f vl-map.js && git commit --amend --no-edit && git pull",
+    "release:prepare": "npm run release:prepare:build && npm run release:prepare:commit",
```

Comment:
Momenteel gaat er tijdens de release iets anders gebeuren dan wat we vroeger deden. Rollup wordt niet langer aangeroepen (of het resultaat wordt overschreven). Rollup regelde het samenvoegen van alle individuele source files tot 1 grote file en het importeerbaar maken van de css als string.
Dat is nu niet meer het geval, dus de gebuilde .js en .src.js files zien er nu helemaal anders uit. Of we nu willen dat de individuele source files samengevoegd worden tot 1 file zoals voorheen gebeurde of niet, laat ik aan @coemans  over. Maar het gaat momenteel niet werken zo aangezien de importpaden niet kloppen.